vendor/strauss/lucatume/di52/aliases.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Registers the library aliases redirecting calls to the `tad_DI52_`, non-namespaced, class format to the namespaced
+ * classes.
+ *
+ * @license GPL-3.0
+ * Modified by StellarWP on 28-August-2022 using Strauss.
+ * @see https://github.com/BrianHenryIE/strauss
+ */
+
+$aliases = [
```

Comment:
The `class_alias` function will pre-load the aliased classes when you call it. Not an issue in this specific application, but something to note/share when eager-loading of the classes is not a desirable side effect. The solution to keep lazy-loading aliased classes is to do so in an autoload function.